content/blog/2025/251028-stac-sprint-eu-recap.md

```diff
@@ -0,0 +1,77 @@
+---
+date: 2025-10-28T08:00:00-00:00
+title: "STAC+Zarr Community Sprint 2025 Recap"
+author: "[Emmanuel Mathot](https://www.linkedin.com/in/emmanuelmathot/)"
+author_title: "Europe Partnership Lead & Solutions Engineer at Development Seed, STAC PSC Member"
+summary: "The first European STAC sprint brought together developers at ESA ESRIN to establish practical patterns and best practices for integrating STAC with Zarr for cloud-native multidimensional geospatial data."
+---
+
+From October 14-16, the STAC and Zarr communities came together at ESA ESRIN in Frascati, Italy, for the first-ever STAC sprint in Europe focused on advancing cloud-native multidimensional geospatial data. After three intensive days of collaborative development, we're excited to share what we accomplished and the path forward for STAC-Zarr integration.
+
+The only thing that I think is ‘off’ in the post is the timeline that says GeoParquet 1.1 was ‘published three months after the introduction of `GEOMETRY` and `GEOGRAPHY` in Parquet’. GeoParquet 1.1 was June 19th of 2024, while the geospatial types landed in Parquet core in March of 2025.
```

Comment:
Looks like this got copy-pasted in by mistake

Reply:
indeed. Thx!